docs/developers.html

```diff
@@ -0,0 +1,209 @@
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Upgrade Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Administration Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Virtual Machine Management Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Introduction to the Administration Portal</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Introduction to the VM Portal</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Self-Hosted Engine Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Data Warehouse Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Metrics Store Installation Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Metrics Store User Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Disaster Recovery Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Gluster Hyperconvergence Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Orb</a></li>
+        <li class="ov-nav-main__sub-item ov-nav-main__sub-item--heading">Release notes</li>
+        <li class="ov-nav-main__sub-item">Latest Release Notes</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Data Warehouse Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Metrics Store Installation Guide</a></li>
+        <li class="ov-nav-main__sub-item">Older Release Notes</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Data Warehouse Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Metrics Store Installation Guide</a></li>
+      </ul>
+    </li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="developers.html">Developers</a>
+    </li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="community.html">Community</a></li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="https://lists.ovirt.org/archives/">Forum</a></li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="blog.html">Blog</a></li>
+  </ul>
+</nav>
+  </header>
+<br>
+<br>
+   <section class="ov-site-section ov-masthead--community">
```

Comment:
let's add a `<main>` wrapper to this page. We can then style it with grid to get the layout you are looking for. We should make sure there is a main tag on every page (good a11y).